src/js/index.js

```diff
@@ -33,7 +35,11 @@ window.onload = () => {
   editTask();
   // adding new task
   addNewTask();
```

Comment:
[Optional] Whenever a new task is added to the array, the page is refreshed. I strongly suggest your prevent page refreshes when adding a new task to improve the user experience👍